Dashboard/app/js/lib/controllers/permissions.js

```diff
@@ -36,106 +37,106 @@ FLOW.permControl = Ember.Controller.create({
 
   /* query based on survey (group) ancestorIds whether a user has
   permissions for data deletion */
-  canDeleteData: function(ancestorIds) {
-      var pathPermissions = FLOW.currentUser.get('pathPermissions');
-      var canDelete = false;
-      ancestorIds.forEach(function(id){
-          if(id in pathPermissions && pathPermissions[id].indexOf("DATA_DELETE") > -1) {
-              canDelete = true;
-          }
-      });
-      return canDelete;
-    },
+  canDeleteData(ancestorIds) {
+    const pathPermissions = FLOW.currentUser.get('pathPermissions');
+    let canDelete = false;
+    ancestorIds.forEach((id) => {
+      if (id in pathPermissions && pathPermissions[id].indexOf('DATA_DELETE') > -1) {
+        canDelete = true;
+      }
+    });
+    return canDelete;
+  },
 
   /* takes a survey (ember object) and checks whether the current user
     has edit permissions for the survey */
-  canEditSurvey: function(survey) {
-    var permissions;
+  canEditSurvey(survey) {
+    let permissions;
     if (!Ember.none(survey)) {
       permissions = this.permissions(survey);
     }
-    return permissions && permissions.indexOf("PROJECT_FOLDER_UPDATE") > -1;
+    return permissions && permissions.indexOf('PROJECT_FOLDER_UPDATE') > -1;
   },
 
   /* takes a form (ember object) and checks with user permissions
   whether the current user has edit permissions for the form */
-  canEditForm: function(form) {
-    var permissions;
+  canEditForm(form) {
+    let permissions;
     if (!Ember.none(form)) {
       permissions = this.permissions(form);
     }
-    return permissions && permissions.indexOf("FORM_UPDATE") > -1;
+    return permissions && permissions.indexOf('FORM_UPDATE') > -1;
   },
 
-  canEditResponses: function (form) {
-    var permissions;
+  canEditResponses(form) {
+    let permissions;
     if (!Ember.none(form)) {
       permissions = this.permissions(form);
     }
-    return permissions && permissions.indexOf("DATA_UPDATE") > -1;
+    return permissions && permissions.indexOf('DATA_UPDATE') > -1;
   },
 
-  canManageDevices: Ember.computed(function () {
-    var currentUserPermissions = FLOW.currentUser.get('pathPermissions');
-    for (var perms in currentUserPermissions) {
-      if (currentUserPermissions[perms].indexOf("DEVICE_MANAGE") > -1) {
+  canManageDevices: Ember.computed(() => {
```

Comment:
In some cases I see `function () {}` replaced like here and in other cases not...

Reply:
@muloem can you show me an example?